docs/v1/index.html

```diff
@@ -2594,6 +2806,88 @@
     </h2>
 
     <p>
+      <div class="anchor" id="1.12.0"></div>
+<b class="header">
+  <a href="https://github.com/jashkenas/coffeescript/compare/1.11.1...1.12.0">1.12.0</a>
+  <span class="timestamp"> &mdash; <time datetime="2016-12-01">December 1, 2016</time></span>
+</b>
+      <ul>
+        <li>
+          CoffeeScript now supports ES2015
+          <a href="#tagged-template-literals">tagged template literals</a>.
+          Note that using tagged template literals in your code
+          makes you responsible for ensuring that either your runtime supports
+          tagged template literals or that you transpile the output JavaScript
+          further to a version your target runtime(s) support.
+        </li>
+        <li>
+          CoffeeScript now provides a
+          <a href="#generator-iteration"><code>for…from</code></a>
+          syntax for outputting ES2015
+          <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of">
+          <code>for…of</code></a>. (Sorry they couldn’t match,
+          but we came up with <code>for…of</code> first for something
+          else.) This allows iterating over generators or any other iterable
+          object. Note that using <code>for…from</code> in your code
+          makes you responsible for ensuring that either your runtime supports
+          <code>for…from</code> or that you transpile the output
+          JavaScript further to a version your target runtime(s) support.
+        </li>
+        <li>
+          Triple backticks (<code>```</code>) allow the creation of embedded
+          JavaScript blocks where escaping single backticks is not required,
+          which should improve interoperability with ES2015 template literals
+          and with Markdown.
+        </li>
+        <li>
+          Within single-backtick embedded JavaScript, backticks can now be
+          escaped via <code>\`</code>.
+        </li>
+        <li>
+          CoffeeScript’s <a href="#objects_and_arrays">shorthand object
+          notation</a> is finally documented.
+        </li>
```

Comment:
In my opinion, all the documentation or build system improvement points (such as this one) are just noise in the change log. It’s really awesome work, but not needed to be in the change log. Again, that’s _my_ opinion. :)